miso-web/src/spring-test/java/uk/ac/bbsrc/tgac/miso/webapp/springtest/NoteRestControllerST.java

```diff
@@ -83,16 +83,18 @@ public void testDeleteNote() throws Exception {
 
     lib = currentSession().get(LibraryImpl.class, 110005);
     int sizeAfter = lib.getNotes().size();
-    assertTrue(sizeBefore == sizeAfter + 1);
+    assertEquals(sizeBefore - 1, sizeAfter);
+    assertFalse(lib.getNotes().stream().anyMatch(x -> x.getId() == 1));
+
   }
 
   @Test
   public void testBulkDeleteNote() throws Exception {
     BulkDeleteRequest req = new BulkDeleteRequest("Library", 110005L, Arrays.asList(1L, 3L));
 
     LibraryImpl lib = currentSession().get(LibraryImpl.class, 110005);
-    assertFalse(lib.getNotes().isEmpty());
-
+    assertTrue(lib.getNotes().stream().anyMatch(x -> x.getId() == 1));
+    assertTrue(lib.getNotes().stream().anyMatch(x -> x.getId() == 3));
```

Comment:
Assert the original size too, to ensure there aren't additional attachments being deleted